pkg/apis/replication/v1/conditions.go

```diff
@@ -31,6 +31,17 @@ type ConditionType string
 const (
 	// ConditionTypeConfigured indicates that the replication has been configured.
 	ConditionTypeConfigured ConditionType = "Configured"
+	// ConditionTypeEnsuredInSync indicates that the replication consistency was checked.
+	ConditionTypeEnsuredInSync ConditionType = "EnsuredInSync"
+	// ConditionTypeAborted indicates that the replication was canceled with abort option.
+	ConditionTypeAborted ConditionType = "Aborted"
+
+	// ConditionTypeConfiguredActive describes synchronization as active.
+	ConditionTypeConfiguredActive = "Active"
```

Comment:
I think these constants (active, inactive, invalid) should named "ConditionReasonXXXYYY".
Because currently it might be confused with ConditionType

Reply:
changed